src/Symfony/Bundle/FrameworkBundle/Tests/Command/AboutCommand/AboutCommandTest.php

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand;
+
+use Symfony\Bundle\FrameworkBundle\Command\AboutCommand;
+use Symfony\Bundle\FrameworkBundle\Console\Application;
+use Symfony\Bundle\FrameworkBundle\Tests\Command\AboutCommand\Fixture\TestAppKernel;
+use Symfony\Bundle\FrameworkBundle\Tests\TestCase;
+use Symfony\Component\Console\Tester\CommandTester;
+use Symfony\Component\Filesystem\Exception\IOException;
+use Symfony\Component\Filesystem\Filesystem;
+
+class AboutCommandTest extends TestCase
+{
+    /** @var TestAppKernel */
+    private $kernel;
+    /** @var Filesystem */
+    private $fs;
+
+    protected function setUp(): void
+    {
+        $this->fs = new Filesystem();
+        $this->kernel = new TestAppKernel('test', true);
+        $this->fs->mkdir($this->kernel->getProjectDir());
+    }
+
+    protected function tearDown(): void
+    {
+        try {
+            $this->fs->remove($this->kernel->getProjectDir());
+        } catch (IOException $e) {
+        }
+    }
+
+    public function testAboutWithReadableFiles()
+    {
+        $this->fs->dumpFile($this->kernel->getCacheDir().'/readable_file', str_repeat('XXX', 1000000));
```

Comment:
If the actual file contents are not important, we could slightly improve test performance if we just do something like this:

```php
$this->fs->dumpFile($this->kernel->getCacheDir().'/readable_file', 'The file content.');
```

Reply:
Done